data/initrd/scripts/url/hmc/menu

```diff
@@ -0,0 +1 @@
+HMC
```

Comment:
In fact C == Console, but I'd rather see it too long as not everyone using it IS actually an enthusiast (QA, ourselves, ...)